setup.py

```diff
@@ -14,7 +14,7 @@
              'configargparse',
              'prometheus-client',
              #'systemd', ??? Unknown which module this is
-             'haproxy-log-analysis<2.0',
+             'haproxy-log-analysis',
```

Comment:
Should be `haproxy-log-analysis>=2.0`, since older versions of `haproxy-log-analysis`won't work with the changes in this commit.